### DISCUSSION TITLE
Why talos state become not ready?

### DISCUSSION THREAD

**Discussion by O-corn** [siderolabs/talos#7753] [Q&A]
I created a pod with one container included, and ran a workload in it. After a while, the state of talos turn to **READY:false**, and it removed my ip. Few minutes later, the state of kubelet, Apiserver, controller-manage, scheduler become **n/a**. Could you give me some hints to debug on this problem?
![image](https://github.com/siderolabs/talos/assets/44917389/fd5808d2-731a-4c0c-816e-4b9d6d6a7c52)
![image](https://github.com/siderolabs/talos/assets/44917389/70f3a997-880b-4576-b6e0-c5ec44bdf568)
![image](https://github.com/siderolabs/talos/assets/44917389/0f55fa65-9146-4274-86ed-87cfdcfbeb62)
![image](https://github.com/siderolabs/talos/assets/44917389/73ff0ba9-d460-40c5-813b-bc4568008006)

**Comment by O-corn** [comment]
When state of kubelet, Apiserver, controller-manage, scheduler become n/a, I noticed that the workload is still running, because from the console I can see the cpu usage is still high.
![image](https://github.com/siderolabs/talos/assets/44917389/cdcfdce2-3b0d-48f1-a228-7e5e7f3c32d4)

**Comment by smira** [comment]
Please submit the text version of the logs, preferably output of talosctl support.

**Reply by O-corn** [reply]
After the system crashed, I cannot communicate with the system by talosctl(since the ip is removed, don't know the reason), so **talosctl dmesg** got stuck..... I can only see the logs on the screen...... or do you have some hints on getting the text version of system logs?

**Reply by smira** [reply]
You have two IPs listed on your screenshot, where do they come from?

**Reply by smira** [reply]
There's no way we could help without you providing some information, you can reboot your machine and run `talosctl support`.
